gdx/src/com/badlogic/gdx/graphics/g3d/utils/shapebuilders/RenderableShapeBuilder.java

```diff
@@ -136,71 +137,73 @@ public static void buildNormals (MeshPartBuilder builder, Renderable renderable,
 			verticesQuantity = renderable.meshPart.size;
 		}
 
-		ensureVerticesCapacity(verticesQuantity * attributesSize);
```

Comment:
Why did you remove `ensureVerticesCapacity`?

Reply:
@dar-dev

Reply:
it sas used for static re-allocation. No longer used.